### PR TITLE
Migrate to microservice repository template

### PR DESCRIPTION
**Pull request by lkuchenb**

### CODE REVIEW

File: .readme_template.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+
```
Comment (KerstenBreuer) [inline .readme_template.md:1]: If we are not using the readme template from the template repo, there is little value in having the readme generated at all. It would be easier to just write the Readme directly.

File: .devcontainer/dev_launcher [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# adapt to package name
+ghga-datasteward-kit "$@"
```
Comment (KerstenBreuer) [inline .devcontainer/dev_launcher:4]: I don't think we need this. Is not a service that shall be launched.

File: .devcontainer/docker-compose.yml [hunk 3]
```diff
@@ -24,12 +26,18 @@ services:
...
 
+    # Uncomment the next line to use a non-root user for all processes.
     user: vscode
-    # Use "forwardPorts" in **devcontainer.json** to forward an app port locally.
-    # (Adding the "ports" property to this file will not forward from a Codespace.)
 
 
 
+    # Please remove service dependencies that are not needed:
+  mongodb:
```
Comment (KerstenBreuer) [inline .devcontainer/docker-compose.yml:35]: Are we using mongo?

File: .github/workflows/pypi_publish.yaml [hunk 4]
```diff
@@ -1,72 +0,0 @@
-name: PyPI Publish
```
Comment (KerstenBreuer) [inline .github/workflows/pypi_publish.yaml:1]: Why was this deleted? We want to publish to PyPI, right?

File: pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+classifiers = [
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.9",
+    "License :: OSI Approved :: Apache Software License",
+    "Topic :: Internet :: WWW/HTTP :: HTTP Servers",
+    "Topic :: Scientific/Engineering :: Bio-Informatics",
+]
+dependencies = [
+    "crypt4gh==1.6",
+    "hexkit[s3]>=0.10,<0.11",
```
Comment (KerstenBreuer) [inline pyproject.toml:25]: Isn't it compatible with the latest version?
```suggestion
    "hexkit[s3]~=0.11",
```

File: pyproject.toml [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+license = { text = "Apache 2.0" }
+classifiers = [
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.9",
+    "License :: OSI Approved :: Apache Software License",
+    "Topic :: Internet :: WWW/HTTP :: HTTP Servers",
+    "Topic :: Scientific/Engineering :: Bio-Informatics",
+]
+dependencies = [
+    "crypt4gh==1.6",
```
Comment (KerstenBreuer) [inline pyproject.toml:24]: Any reason to pin?
```suggestion
    "crypt4gh >=1.6, <2",
```

File: pyproject.toml [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    "hexkit[s3]>=0.10,<0.11",
+    "ghga-transpiler>=1.2.0,<2.0",
+    "metldata>=0.4.1,<0.5",
```
Comment (KerstenBreuer) [inline pyproject.toml:27]:
```suggestion
    "metldata~=0.4.1",
```

**Review by KerstenBreuer — CHANGES_REQUESTED** [review]
Looks good just a few suggestions.

**Comment by lkuchenb** [comment]
@KerstenBreuer Thanks for your feedback

**Review by KerstenBreuer — APPROVED** [review]
